### ISSUE TITLE
Feature request: Capture wrong grains.get

### ISSUE THREAD

**Issue by jbouter** [warpnet/salt-lint#58]
I'd like to see a function implemented where usage of wrong `grains.get` is detected and reported.

For example:
**wrong**:
```code
grains.get('grain:item')
```

**good**:
```code
salt['grains.get']('grain:item')
```

```code
grains['grain']['item']
```
